frontend/public/components/image-stream.tsx

```diff
@@ -85,23 +85,29 @@ const { common } = Kebab.factory;
 const menuActions = [...common];
 
 const ImageStreamTagsRow: React.SFC<ImageStreamTagsRowProps> = ({imageStream, specTag, statusTag}) => {
+  const imageStreamStatus = _.get(imageStream, 'status');
   const latest = _.get(statusTag, ['items', 0]);
   const from = _.get(specTag, 'from');
   const referencesTag = _.get(specTag, 'from.kind') === 'ImageStreamTag';
   const image = _.get(latest, 'image');
   const created = _.get(latest, 'created');
+  const dockerRepositoryCheck = _.has(imageStream, 'metadata.annotations.openshift.io/image.dockerRepositoryCheck');
```

Comment:
Does this actually work? The expression looks wrong because of the special characters.

Reply:
yes the special chars were a problem. Fixed now